proposals/lip-0054.md

```diff
@@ -275,9 +285,9 @@ def execute(trs: Transaction) -> None:
             "value": sha256(emptyStore),
             "bitmap": entry.bitmap
         }
-        storeQueries.append(query)
+        storeQueriesUpdate.append(query)
 
-    terminatedStateAccount(trsParams.chainID).stateRoot = SMTCalculateRoot(trsParams.siblingHashes, storeQueries)
+    terminatedStateAccount(trsParams.chainID).stateRoot = SMTCalculateRoot(trsParams.siblingHashes, storeQueriesUpdate)
 ```
 
 #### Recover Function
```

Comment:
Shouldn't we also remove the inclusion proof of Message Recovery Command verification (lines 408-421)?

Reply:
I think these are regular Merkle inclusion proofs. In principle we should also remove those if it is possible to force a node to run the verification

Reply:
moved to execution

Reply:
cbr